### PR TITLE
Scala 2.13 support for 2.x

### PR DESCRIPTION
**Pull request by solicode**
Surprisingly the only breaking change I ran into was `deep` being removed in 2.13 collections.

A few of the tests don't pass locally for me, but I think that might be due to implementation differences in anti-aliasing and so on with the JDK I'm using? My environment probably doesn't match the environment that was used to generate `/com/sksamuel/scrimage/canvas/drawablestring.png` and so on. Anyway, let's see how those run in CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
+dist: trusty
 language: scala
 scala:
    - 2.10.5
    - 2.11.7
-   - 2.12.0-M3
+   - 2.12.10
+   - 2.13.0
```
Comment (manfredi-giordano) [inline .travis.yml]: Shouldn't it be `2.13.1`?

Comment (solicode) [reply .travis.yml]: I can try that now. Last time I tried, it wasn't available yet.

Comment (solicode) [reply .travis.yml]: Cool, that worked.

**Comment by sksamuel** [comment]
Merged, the auto snapshot should release it.

**Comment by solicode** [comment]
Thanks @sksamuel. From what I can tell, the auto snapshot release is only for 3.x (snapshot_deploy.sh). I don't think it applies for 2.1.x.

**Comment by rechemendia** [comment]
Guys, sorry for the ignorance, how can I consume the version compatible with scala 2.13 from sbt, I tried it in many ways and I couldn't /-:

**Comment by solicode** [comment]
@rechemendia The PR has been merged, but it looks like a new version hasn't been published yet.

**Comment by sksamuel** [comment]
A new release is on my agenda for the weekend.

**Comment by rechemendia** [comment]
@sksamuel any news about the publication? thx

**Comment by woutermont** [comment]
Anxiously waiting for this yumminess to appear on mvn  ...

**Comment by csaltos** [comment]
@sksamuel thank you very much for your code and your time, we appreciate it very much ... may you help us publishing the new 2.1.x version for Scala 2.13 with this PR ? ... thank you !! 👍 😎

**Pull request merged**